pages.pt_BR/common/zip.md

```diff
@@ -3,30 +3,34 @@
 > Ferramenta de compressão de arquivos em arquivos zip.
 > Mais informações: <https://manned.org/zip>.
 
-- Compactando arquivos em um arquivo zip:
+- Compacta arquivos:
 
-`zip {{output.zip}} {{arquivo1}} {{arquivo2}} {{arquivo3}}`
```

Comment:
It's better to use `caminho/para/compactado.zip` (google suggest this translation) because you can write path to archive but not just archive name. The same applies for other examples.